build/config/BUILDCONFIG.gn

```diff
@@ -291,7 +291,12 @@ if (is_cobalt) {
   if (custom_toolchain != "") {
     cobalt_toolchain = custom_toolchain
   } else if (_default_toolchain != "") {
-    cobalt_toolchain = _default_toolchain
+    if (target_os == "linux" && is_clang) {
+      cobalt_toolchain =
```

Comment:
The values of the host_toolchain and cobalt_toolchain could be the still the same, right?

Reply:
the value will be same, the label will different enabling me to use current_toolchain == cobalt_toolchain to add configs only while building with cobalt_toolchain and not with host_toolchain

Note, in some cases valuee might be different when i try to build with current linux args.gn file, the 2 values match